spec.bs

```diff
@@ -177,9 +177,15 @@ are:
   1. TODO: document .well-known fetches for cross-origin joins.
   1. [=Queue a task=] to [=resolve=] |p| with `undefined`.
   1. If the browser is currently storing an interest group with `owner` and `name` that matches

```

Comment:
It might be good follow up work to define a function / dfn to look up stored IGs and call that here, something like `Let |ig| be the result of [=looking up an interest group=] with |owner| and |name|`, etc. And maybe a dfn for "removing an interest group". But that seems fine as follow-up refactoring work.